exporter/stackdriverexporter/config.go

```diff
@@ -26,6 +28,9 @@ type Config struct {
 	Endpoint                      string                   `mapstructure:"endpoint"`
 	NumOfWorkers                  int                      `mapstructure:"number_of_workers"`
 	SkipCreateMetricDescriptor    bool                     `mapstructure:"skip_create_metric_descriptor"`
+	BundleDelayThreshold          time.Duration            `mapstructure:"bundle_delay_threshold"`
+	BundleCountThreshold          int                      `mapstructure:"bundle_count_threshold"`
```

Comment:
Exposing Bundler options here seems clunky. There's no common way to express these across exporters, but the most similar are Azure Monitor https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2f1c4e39c98179fcbe4b99e724f75bc6c285ab6a/exporter/azuremonitorexporter/config.go#L29-L30
and Kinesis https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2f1c4e39c98179fcbe4b99e724f75bc6c285ab6a/exporter/kinesisexporter/config.go#L52-L54
Something like
```go
FlushInterval        time.Duration `mapstructure:"flush_interval"`
MaxBytesPerBatch     int           `mapstructure:"max_bytes_per_batch"`
```

Reply:
Oops, those aren't the same thing 🤔 
```go
MaxSpansPerBatch int `mapstructure:"max_spans_per_batch"`
```

Reply:
in general I don't think the exporters *should* have batch sizes and the batch + queued_processor should handle it.  It will require a rewrite of the exporter I think though.

Reply:
stackdriver exporter uses PushMetricsProto api to export metrics proto. This api does notuse the bundler. See this [comment](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/626e69eeee9db2d84284abdbfe6f2c27d37d74e8/stackdriver.go#L384-L394). So batch+queued_processor is the right approach and shouldn't require any changes in the exporter.

Reply:
I don't think this is the case for traces.

Reply:
You are right about traces. I think it would be better to update the exporter to accept a batch of spans instead.

Reply:
https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/pull/249 would appreciate if you could have a look.  We're currently experiencing span loss on the exporter due to buffering in production so it would be nice to fix it rather than just turn it off.

Reply:
merged and release [v0.13.0](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/releases/tag/v0.13.0).